OneBusAway_WatchKit_Extension/OBAArrivalsAndDeparturesInterfaceController.m

```diff
@@ -0,0 +1,133 @@
+//
+//  OBAArrivalsAndDeparturesInterfaceController.m
+//  org.onebusaway.iphone
+//
+//  Created by Shannon Young on 4/14/15.
+//  Copyright (c) 2015 OneBusAway. All rights reserved.
+//
+
+#import "OBAArrivalsAndDeparturesInterfaceController.h"
+
+#import "OBAStopWK.h"
+#import "OBAArrivalAndDepartureWK.h"
+
+#import "OBAStopRowController.h"
+
+#define kStopRouteDetailRowType @"Route Detail Row"
+
+@interface OBAArrivalsAndDeparturesInterfaceController ()
+
+@property (nonatomic) NSMutableOrderedSet *arrivalsAndDepartures;
+
+@property (weak, nonatomic) NSURLSessionDataTask *dataTask;
+
+@end
+
+@implementation OBAArrivalsAndDeparturesInterfaceController
+
+- (void)awakeWithContext:(id)context {
+    [super awakeWithContext:context];
+
+    // Setup defaults
+    self.arrivalsAndDepartures = [NSMutableOrderedSet new];
+    self.maxCount = NSIntegerMax;
+    self.minutesBefore = 5;
+    self.minutesAfter = 35;
+}
+
+- (void)didDeactivate {
+    // This method is called when watch view controller is no longer visible
+    [super didDeactivate];
+
+    [self.dataTask cancel];
+    self.dataTask = nil;
+}
+
+- (NSArray *)filteredRouteIds {
+    return nil;
+}
+
+- (void)refreshArrivalsAndDeparturesForStop:(OBAStopWK *)stop {
```

Comment:
@beliyd 
I started out trying to pull it into a library and stopped primarily for two reasons:

1. Currently, the main project is targeted at iOS 6.0. Because of that, I couldn't use a dynamic embedded library.

2. It's already a *huge* change just to add what I considered the bare minimum for an Apple Watch app (that wasn't pointless).  I wanted to keep this iteration separated as much as possible. Towards that end, I tried to keep to the existing model (but simplified) so that it would be fairly straight-forward to change to using a shared library, but used a naming convention that ended with "WK".

In the long run, I would agree that a shared services/model library would be a lot cleaner and easier to maintain, but getting there is more than I had the time to tackle. :)